whois-api/src/test/java/net/ripe/db/whois/api/rest/ZonemasterTestIntegration.java

```diff
@@ -1,19 +1,19 @@
 package net.ripe.db.whois.api.rest;
 
 import net.ripe.db.whois.api.AbstractIntegrationTest;
-import net.ripe.db.whois.api.ZonemasterDummy;
+import net.ripe.db.whois.api.zonemasterservice.ZonemasterDummy;
 import net.ripe.db.whois.common.rpsl.RpslObject;
```

Comment:
Just a suggestion , feel free to ignore. If you want to refactor use package name with just net.ripe.db.whois.api.zonemaster? or net.ripe.db.whois.api.zonemaster.service?

Reply:
oh, I did it when we had the two objects, now we just the service we could leave as before